moveit_core/planning_scene/src/planning_scene.cpp

```diff
@@ -1465,6 +1466,8 @@ void PlanningScene::removeAllCollisionObjects()
       world_->removeObject(object_id);
       removeObjectColor(object_id);
       removeObjectType(object_id);
+      // TODO: How about octomap?
```

Comment:
worth a separate issue here?

Reply:
After thinking about it, I think we shouldn't handle it since there is a reserved name for it and user can't add an object with the same name